manim/animation/creation.py

```diff
@@ -274,19 +274,28 @@ class Write(DrawBorderThenFill):
         class ShowWrite(Scene):
             def construct(self):
                 self.play(Write(Text("Hello").scale(3)))
+
+    .. manim:: ShowWriteReversed
+
+        class ShowWriteReversed(Scene):
+            def construct(self):
+                self.play(Write(Text("Hello").scale(3), reverse=True))
     """
 
     def __init__(
         self,
         vmobject: Union[VMobject, OpenGLVMobject],
         rate_func: Callable[[float], float] = linear,
+        reverse: bool = False,
         **kwargs,
     ) -> None:
         run_time: Optional[float] = kwargs.pop("run_time", None)
         lag_ratio: Optional[float] = kwargs.pop("lag_ratio", None)
         run_time, lag_ratio = self._set_default_config_from_length(
             vmobject, run_time, lag_ratio
         )
+        self.reverse = reverse
+        self.vmobject = vmobject
```

Comment:
That shouldn't be necessary, animations store the mobject they are manipulating as `self.mobject` IIRC.

Reply:
That works, thanks!